packages/app/src/extension/background/vault-manager.ts

```diff
@@ -122,7 +121,7 @@ export const vaultReducer = async (message: MessageFromApp): Promise<Vault> => {
           gaiaHubConfig,
         });
       };
-      void updateConfig();
+      await updateConfig();
```

Comment:
I'm not sure I want this to be blocking. It doesn't change vault state at all, and even if UI was immediately closed, this happens in the background script so we don't need to worry about prematurely cutting this off. Was there a different reason?

Reply:
This was the result of trying to debug why new accounts don't show up instantly once this has completed. There seems to be a bug such that the wallet config does not update after a new account is created. This change didn't actually affect anything as far as I can tell.